app/components/mandatarissen/update-state.js

```diff
@@ -52,19 +50,8 @@ export default class MandatarissenUpdateState extends Component {
         await this.args.mandataris.bekleedt
       ).bevatIn
     ).slice();
-    this.bestuursorganenForFractie =
-      await this.tijdsspecialisaties.getCurrentTijdsspecialisaties(
-        this.store,
-        this.bestuurseenheid,
-        {
-          startDate: moment(
-            this.bestuursorganenOfMandaat[0].bindingStart
-          ).format('YYYY-MM-DD'),
-          endDate: moment(this.bestuursorganenOfMandaat[0].bindingEinde).format(
-            'YYYY-MM-DD'
-          ),
-        }
-      );
+    this.bestuursperiode =
+      await this.bestuursorganenOfMandaat[0].heeftBestuursperiode;
```

Comment:
Do we need to catch if no bestuursperiode is found? First check if the item on index 0 is there?

Reply:
Shouldn't really be possible, but can't hurt to check.